binder.go

```diff
@@ -229,6 +231,10 @@ func bindSlice(params *Params, name string, typ reflect.Type) reflect.Value {
 
 		// Handle the indexed case.
 		if index > -1 {
+			// Just ignore illegal index, fix issue #1424
+			if index > maxIndexBound {
+				return
```

Comment:
Could you a a log statement here like
```suggestion
				binderLog.Error.Println("Invalid parameter index, ignoring parameter","index", maxIndex,"key",key)
				return
```